_pytest/skipping.py

```diff
@@ -230,7 +230,8 @@ def pytest_runtest_makereport(item, call):
     if hasattr(item, '_unexpectedsuccess') and rep.when == "call":
         # we need to translate into how pytest encodes xpass
         rep.wasxfail = "reason: " + repr(item._unexpectedsuccess)
-        rep.outcome = "failed"
+        # TODO: Do we need to check for strict xfail here as well?
```

Comment:
I suggest creating a helper function:

```python
def is_strict_xfail(item, marker):
    strict_default = item.config.getini('xfail_strict')
    return evalxfail.get('strict', strict_default)
```

And reusing in both places.

I also noticed you can reuse this same function in `check_strict_xfail` in the same module.